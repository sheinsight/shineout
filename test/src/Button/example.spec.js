import exampleTest from '../../example'

describe('Button[snapshot]', () => {
  exampleTest('Button')
})
