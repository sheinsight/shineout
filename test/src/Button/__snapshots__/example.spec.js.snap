// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Button[site/pages/components/Button/example-1-base.js] 1`] = `
<div>
  <button>
    <span>
      Default
    </span>
  </button>
  <button>
    <span>
      Primary
    </span>
  </button>
  <button>
    <span>
      Secondary
    </span>
  </button>
  <button>
    <span>
      Success
    </span>
  </button>
  <button>
    <span>
      Warning
    </span>
  </button>
  <button>
    <span>
      Danger
    </span>
  </button>
  <button>
    <span>
      Link
    </span>
  </button>
</div>
`;

exports[`Snapshot test: Button[site/pages/components/Button/example-1-text.js] 1`] = `
<div>
  <button>
    <span>
      Default
    </span>
  </button>
  <button>
    <span>
      Primary
    </span>
  </button>
  <button>
    <span>
      Secondary
    </span>
  </button>
  <button>
    <span>
      Success
    </span>
  </button>
  <button>
    <span>
      Warning
    </span>
  </button>
  <button>
    <span>
      Danger
    </span>
  </button>
  <button>
    <span>
      Link
    </span>
  </button>
</div>
`;

exports[`Snapshot test: Button[site/pages/components/Button/example-2-icon.js] 1`] = `
<div>
  <button>
    <i />
    <span>
      left
    </span>
  </button>
  <button>
    <span>
      right
    </span>
    <i />
  </button>
  <button>
    <span>
      ce
    </span>
    <i />
    <span>
      ter
    </span>
  </button>
</div>
`;

exports[`Snapshot test: Button[site/pages/components/Button/example-3-size.js] 1`] = `
<div>
  <div>
    <button>
      <span>
        Default
      </span>
    </button>
    <button>
      <span>
        Primary
      </span>
    </button>
    <button>
      <span>
        Secondary
      </span>
    </button>
    <button>
      <span>
        Success
      </span>
    </button>
    <button>
      <span>
        Warning
      </span>
    </button>
    <button>
      <span>
        Danger
      </span>
    </button>
    <button>
      <span>
        Link
      </span>
    </button>
  </div>
  <br />
  <div>
    <button>
      <span>
        Default
      </span>
    </button>
    <button>
      <span>
        Primary
      </span>
    </button>
    <button>
      <span>
        Secondary
      </span>
    </button>
    <button>
      <span>
        Success
      </span>
    </button>
    <button>
      <span>
        Warning
      </span>
    </button>
    <button>
      <span>
        Danger
      </span>
    </button>
    <button>
      <span>
        Link
      </span>
    </button>
  </div>
  <br />
  <div>
    <button>
      <span>
        Default
      </span>
    </button>
    <button>
      <span>
        Primary
      </span>
    </button>
    <button>
      <span>
        Secondary
      </span>
    </button>
    <button>
      <span>
        Success
      </span>
    </button>
    <button>
      <span>
        Warning
      </span>
    </button>
    <button>
      <span>
        Danger
      </span>
    </button>
    <button>
      <span>
        Link
      </span>
    </button>
  </div>
</div>
`;

exports[`Snapshot test: Button[site/pages/components/Button/example-4-disabled.js] 1`] = `
<div>
  <button>
    <span>
      Default
    </span>
  </button>
  <button>
    <span>
      Primary
    </span>
  </button>
  <button>
    <span>
      Secondary
    </span>
  </button>
  <button>
    <span>
      Success
    </span>
  </button>
  <button>
    <span>
      Warning
    </span>
  </button>
  <button>
    <span>
      Danger
    </span>
  </button>
  <button>
    <span>
      Link
    </span>
  </button>
</div>
`;

exports[`Snapshot test: Button[site/pages/components/Button/example-5-loading.js] 1`] = `
<div>
  <button>
    <span>
      <div />
    </span>
    <span>
      Small
    </span>
  </button>
  <button>
    <span>
      <div />
    </span>
    <span>
      Default
    </span>
  </button>
  <button>
    <span>
      <div />
    </span>
    <span>
      Large
    </span>
  </button>
</div>
`;

exports[`Snapshot test: Button[site/pages/components/Button/example-6-href.js] 1`] = `
<a>
  Home
</a>
`;

exports[`Snapshot test: Button[site/pages/components/Button/example-7-outline.js] 1`] = `
<div>
  <button>
    <span>
      Primary
    </span>
  </button>
  <button>
    <span>
      Secondary
    </span>
  </button>
  <button>
    <span>
      Success
    </span>
  </button>
  <button>
    <span>
      Warning
    </span>
  </button>
  <button>
    <span>
      Danger
    </span>
  </button>
</div>
`;

exports[`Snapshot test: Button[site/pages/components/Button/example-8-group.js] 1`] = `
<div>
  <div>
    <button>
      <span>
        Left
      </span>
    </button>
    <button>
      <span>
        Center
      </span>
    </button>
    <button>
      <span>
        Right
      </span>
    </button>
  </div>
  <br />
  <div>
    <button>
      <span>
        Left
      </span>
    </button>
    <button>
      <span>
        Center
      </span>
    </button>
    <button>
      <span>
        Right
      </span>
    </button>
  </div>
  <br />
  <div>
    <button>
      <span>
        disabled
      </span>
    </button>
    <button>
      <span>
        Center
      </span>
    </button>
    <button>
      <span>
        Right
      </span>
    </button>
  </div>
  <br />
  <div>
    <button>
      <i />
      <span>
        Left
      </span>
    </button>
    <button>
      <span>
        Center
      </span>
    </button>
    <button>
      <span>
        Right
      </span>
      <i />
    </button>
  </div>
  <br />
  <div>
    <button>
      <span>
        Left
      </span>
    </button>
    <button>
      <span>
        Center
      </span>
    </button>
    <button>
      <span>
        Right
      </span>
    </button>
  </div>
</div>
`;
